enterprise_knowledge_retriever/tests/ekr_test.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""
+Enterprise Knowledge Retriever (EKR) Test Script
+
+This script tests the functionality of the Enterprise Knowledge Retriever using unittest.
+It parses documents using the SambaParse service, creates a vector store, and tests the question-answering capabilities.
+
+Usage:
+    python tests/ekr_test.py
+
+Returns:
+    0 if all tests pass, or a positive integer representing the number of failed tests.
+"""
+
+import os
+import sys
+import shutil
+import time
+import unittest
+import logging
+from typing import List, Dict, Any
+
+# Setup logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+# Setup paths
+current_dir = os.getcwd()
+kit_dir = current_dir
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+logger.info(f'kit_dir: {kit_dir}')
+logger.info(f'repo_dir: {repo_dir}')
+
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
+
+from src.document_retrieval import DocumentRetrieval
+from utils.parsing.sambaparse import parse_doc_universal
+
+CONFIG_PATH = os.path.join(kit_dir, 'config.yaml')
+PERSIST_DIRECTORY = os.path.join(kit_dir, "tests", "vectordata", "my-vector-db")
+TEST_DATA_PATH = os.path.join(kit_dir, "tests", "data", "test")
```

Comment:
we may need to update the test_data_path if we change the location of the data/ folder one level up